api/document/js/__tests__/serialize-doc.test.ts

```diff
@@ -44,6 +45,14 @@ describe('serializeDoc()', () => {
     });
   });
 
+  it('converts heading text to section title', () => {
+    const node = factory.sec([
+      factory.heading('Stufff', 1),
+    ]);
```

Comment:
Would it make sense to modify the `sec` factory to take required args for headings?
```
factory.sec('Stuff', 1, [other-children])
```
That'll force us to deal with invalid input from the api, too.

Reply:
I like it! Just filed #983 to deal with that in a separate PR.